headers/types/files/wan.h

```diff
@@ -95,4 +95,34 @@ struct wan_header {
 #pragma pack(pop)
 ASSERT_SIZE(struct wan_header, 10);
 
+struct wan_fragment {
+    int16_t fragment_bytes_index; // negative mean using the previous defined fragment bytes (or to
+                                  // not update it?)
+    int8_t unk1;
+    int8_t unk2;
+
+    // 2 bytes
+    int8_t offset_y : 8;
+    bool unk3 : 1;
+    bool unk4 : 1;
+    uint8_t likely_padding_1 : 3;
+    bool is_mosaic : 1;
+    uint8_t shape_indice : 2; // as used in OBJ attribute 1
```

Comment:
Is this supposed to be `shape_index`?

Reply:
No. indice is to be understood what the dictionary tell me is an obsolete form of indication. I think this terminology in this context come from ppmdu_tools of psy_commando (of which I derive many of my wan knowledge).
I... would prefer to keep it like that.

Reply:
I see. Okay, we'll keep it as is then